src/zeroD/ReactorNet.cpp

```diff
@@ -253,10 +254,55 @@ void ReactorNet::addReactor(Reactor& r)
 void ReactorNet::eval(doublereal t, doublereal* y,
                       doublereal* ydot, doublereal* p)
 {
-    m_time = t; // This will be replaced at the end of the timestep
+    m_time = t; 
     updateState(y);
-    for (size_t n = 0; n < m_reactors.size(); n++) {
-        m_reactors[n]->evalEqs(t, y + m_start[n], ydot + m_start[n], p);
+    m_LHS.assign(m_nv, 1); //m_LHS is a vector representing the coeffcicients on the "left hand side" of each governing equation 
+    m_RHS.assign(m_nv, 0);
+    if (!m_checked_eval_deprecation) {
+        m_have_deprecated_eval.assign(m_reactors.size(), false);
+        for (size_t n = 0; n < m_reactors.size(); n++) {
+            m_reactors[n]->applySensitivity(p);
+            try {
+                m_reactors[n]->evalEqs(t, y + m_start[n], ydot + m_start[n], p);
+                warn_deprecated(m_reactors[n]->type() +
+                    "::evalEqs(double t, double* y , double* ydot, double* params)",
+                    "Reactor time derivative evaluation now uses signature "
+                    "eval(double t, double* ydot)");
+                m_have_deprecated_eval[n] = true;
+            } catch (NotImplementedError& err) {
+                m_reactors[n]->eval(t, m_LHS.data() + m_start[n], m_RHS.data() + m_start[n]);
+                int yEnd = 0;
+                if(n == m_reactors.size()-1){
+                    yEnd = m_RHS.size();
+                } else {
+                    yEnd = m_start[n+1];
+                }
+                    for (int i = m_start[n]; i < yEnd; i++) {
+                     ydot[i] = m_RHS[i]/m_LHS[i];
+                    }
+            }
+            m_reactors[n]->resetSensitivity(p);
+        }
+        m_checked_eval_deprecation = true;
+    } else {
+        for (size_t n = 0; n < m_reactors.size(); n++) {
+            m_reactors[n]->applySensitivity(p);
+            if (m_have_deprecated_eval[n]) {
+                m_reactors[n]->evalEqs(t, y + m_start[n], ydot + m_start[n], p);
+            } else {
+                m_reactors[n]->eval(t, m_LHS.data() + m_start[n], m_RHS.data() + m_start[n]);
+                    int yEnd = 0;
+                if(n == m_reactors.size()-1){
+                    yEnd = m_RHS.size();
+                } else {
```

Comment:
Just a minor formatting tweak:
```suggestion
                if (n == m_reactors.size()-1) {
                    yEnd = m_RHS.size();
                } else {
```
Also applies above to a similar instance in the first branch in this function.